ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -178,17 +186,34 @@
     <Compile Include="Modules\Core\RCTDeviceEventEmitter.cs" />
     <Compile Include="Modules\Core\RCTNativeAppEventEmitter.cs" />
     <Compile Include="Bridge\ReactDelegateFactoryBase.cs" />
+    <Compile Include="Reflection\ExpressionExtensions.cs" />
+    <Compile Include="IReactPackage.cs" />
+    <Compile Include="touch\JSResponderHandler.cs" />
```

Comment:
The capitalization of this folder is inconsistent with the rest of the folders.  All others are camel case starting with a capital letter.